broadside.gemspec

```diff
@@ -19,7 +19,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'activesupport', '>= 3', '< 6'
   spec.add_dependency 'activemodel', '>= 3', '< 6'
-  spec.add_dependency 'aws-sdk', '~> 2.3'
+  spec.add_dependency 'aws-sdk-ecs', '~> 1'
+  spec.add_dependency 'aws-sdk-ec2', '~> 1'
```

Comment:
if we do use this, this should probably pin the major version at least (i.e. use `~> 1.0` or something)

Reply:
actually, that's a good point; isn't this syntax essentially "any major version whatsoever"?

Reply:
yeppp, anything greater or equal to 1
(so technically it excludes pre-1 versions, but after that effectively anything is valid)

Reply:
the SDK docs are confusing about this actually: https://github.com/aws/aws-sdk-ruby/blame/master/README.md#L24 ("_use_ a major version", but uses syntax that doesn't _pin_ a major version).

Reply:
Generally I agree, but their documentation says otherwise:

https://github.com/aws/aws-sdk-ruby#installation

Reply:
O_O

Reply:
I'm not sure if that's a mistake in their documentation because they _do_ use semver (hence the explanation for the 3.0 major bump) ... :/

Reply:
yeah, I'll just use 1.0, aws is always kind of weird about versions (they always have you using `:latest` for their containers also)